--- libmo_unpack-3.1.2.orig/tests/CMakeLists.txt	2015-10-09 07:44:17.000000000 -0300
+++ libmo_unpack-3.1.2/tests/CMakeLists.txt	2018-04-16 16:04:09.877831273 -0300
@@ -1,7 +1,7 @@
 enable_testing()
 find_package(Check REQUIRED)
 include_directories(${CHECK_INCLUDE_DIRS})
-set(LIBS ${LIBS} ${CHECK_LIBRARIES} mo_unpack m rt pthread)
+set(LIBS ${LIBS} ${CHECK_LIBRARIES} mo_unpack m pthread)
 include_directories(. ../src)
 add_executable(test_rle check_rle.c)
 target_link_libraries(test_rle ${LIBS})
